app/views/includes/phase-banner.njk

```diff
@@ -6,6 +6,7 @@
     {% set isLive = currentGatewayAccount.type === "live" %}
     {% set stripeAccountIsSetup = (currentGatewayAccount.connectorGatewayAccountStripeProgress.bankAccount and currentGatewayAccount.connectorGatewayAccountStripeProgress.responsiblePerson
       and currentGatewayAccount.connectorGatewayAccountStripeProgress.vatNumber and currentGatewayAccount.connectorGatewayAccountStripeProgress.companyNumber) %}
+    {% set isRequiresAdditionalKycData = currentGatewayAccount.type === 'live' and currentGatewayAccount.requiresAdditionalKycData and isAdminUser %}
```

Comment:
Should be fine to show for test accounts too. Will make it easy for us to test without having a live account